python/composio/tools/toolset.py

```diff
@@ -809,8 +811,15 @@ def get_action_schemas(
                 actions=remote_actions,
                 tags=tags,
             )
-            for item in remote_items:
-                self.check_connected_account(action=item.name)
+            if check_connected_accounts:
+                for item in remote_items:
+                    self.check_connected_account(action=item.name)
+            else:
+                warnings.warn(
```

Comment:
It's good that a warning is issued when `check_connected_accounts` is set to `False`. However, consider adding more specific information in the warning message about potential consequences or scenarios where this might be useful. This will help users make informed decisions when using this flag.